### DISCUSSION TITLE
Diagnostics Metrics intergation

### DISCUSSION THREAD

**Discussion by sanych-sun** [PrometheusClientNet/Prometheus.Client#217] [Ideas]
There is cool toolset was added into net6 named "Diagnostics Metrics".
https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.metrics

It looks like we have to create a new integration library that will allow easily publish the reported metrics into Prometheus. Should we start a new repo for this?

**Comment by phnx47** [comment]
@sanych-sun Yep, a new repo based on [prom-tmpl](https://github.com/prom-client-net/prom-tmpl)
